packages/protocol/test/bridge/BridgedERC20.test.ts

```diff
@@ -330,7 +330,7 @@ describe("BridgedERC20", function () {
                     .transfer(owner.address, amount)
             )
                 .to.emit(erc20, "Transfer")
-                .withArgs(accountWithTokens.address, owner.address, amount)
+                .withArgs(owner.address, amount)
```

Comment:
i think we do want this argument? This was failing for me locally that the Event was only provided two args but expected 3 on my local tests.

Reply:
cc @cyberhorsey

Reply:
fixed